### DISCUSSION TITLE
I cant change the property

### DISCUSSION THREAD

**Discussion by Tuberian174** [jackvonhouse/custom-node-size#5] [Q&A] [answered]
Hi jack! 

I've tried to use your plugin, but the node_size property doesnt appear. (I'm new to obsidian, so i don't know if i did something wrong...)

![image](https://github.com/user-attachments/assets/8d406c1c-efc0-4aa6-9226-79aef567a368)

I would like some help... 

Thanks for your help!

**Comment by jackvonhouse** [comment, marked as answer]
Hi!
The drop-down list only shows the system tags and the ones you previously used.
So just add `node_size` tag and value in the note properties.
Good luck!

**Reply by johnw3522** [reply]
Hey! I'm also new to obsidian and would like to use your plugin :) 

I appreciate your response to the user above but I still can't get it to work unfortunately. I've tried changing multiple things but I have not figured out what I have done wrong. Maybe I should use a different property type or have I entered the text/value incorrectly? 

I would be very grateful if you could help me out when you have a the time :)

![Skärmavbild 2024-11-09 kl  21 33 47](https://github.com/user-attachments/assets/4f80755e-b050-4d5c-992e-7f3bdc1f7c7b)

**Reply by jackvonhouse** [reply]
@johnw3522 Hi!
The plugin is not automatically enabled after installation.
Go to the community plugins option in settings and enable installed plugin `Custom Node Size`.
After that, the graph should change.
Good luck!

**Reply by johnw3522** [reply]
Hi again! 
Thank you for your reply, appreciate it! 

However I think I have already enabled the plugin correctly
![Skärmavbild 2024-11-11 kl  22 36 19](https://github.com/user-attachments/assets/040d2a37-567f-4fc4-8453-d36772837907)


Here is a picture inside my note
![Skärmavbild 2024-11-11 kl  22 36 38](https://github.com/user-attachments/assets/167af531-68b3-4827-92b3-e38eb81fa751)



Here is a picture of my graph view (There are two lonely nodes that are not connect to any other nodes, my "test" note is the top one of them)
![Skärmavbild 2024-11-11 kl  22 37 01](https://github.com/user-attachments/assets/3ff96413-c970-451d-ae85-eb924e3b3b8c)


Anything else that I've done incorrectly that you notice? :)

**Reply by johnw3522** [reply]
Update: The plugin worked for me after restarting a obsidian a couple of times
